text/0015-market-design-revisit.md

```diff
@@ -0,0 +1,116 @@
+
+In particular, this proposal introduces the following changes:
+- It introduces a `RESERVE_PRICE` that anchors all markets, promoting price synchronicity among it's substitute goods (flexible/renewed/instantanous coretime). 
+    - This reduces complexity.
+    - This makes sure all tenants pay the same for coretime.
+- It reverses the order of the market and renewal phase. 
+    - This allows to fine-tune the price through market forces.
+- It exposes the renewal prices, while still being beneficial for longterm tenants, more to market forces. 
+- It removes the LeadIn period and introduces a (from the perspective of the coretime systemchain) passive Settlement Phase, that allows the secondary market to exert it's force.
+
+The premise of this proposal is to reduce complexity by introducing a common price (that develops releative to capacity consumption of Polkadot UC), while still allowing for market forces to add efficiency. Longterm lease owners still receive priority **IF** they can pay (close to) the market price. This prevents a situation where the renewal price significantly diverges from renewal prices which allows for core captures. While maximum price increase certainty might seem contradictory to efficient price discovery, the proposed model aims to balance these elements, utilizing market forces to determine the price and allocate cores effectively within certain bounds. It must be stated, that potential price increases remain predictable (in the worst-case) but could be higher than in the originally proposed design. The argument remains, however, that we need to allow market forces to affect all prices for an efficient Coretime pricing and allocation.
+
+Ultimately, this the framework proposed here adheres to all requirements stated in RFC-1.
+
+## Stakeholders
+
+Primary stakeholder sets are:
+
+- Protocol researchers and developers, largely represented by the Polkadot Fellowship and Parity Technologies' Engineering division.
+- Polkadot Parachain teams both present and future, and their users.
+- Polkadot DOT token holders.
+
+## Explanation
+
+### Bulk Markets
+ 
+The `BULK_PERIOD` has been restructured into two primary segments: the `MARKET_PERIOD` and `RENEWAL_PERIOD`, along with an auxiliary `SETTLEMENT_PERIOD`. This latter period doesn't necessitate any actions from the coretime system chain, but it facilitates a more efficient allocation of coretime in secondary markets. A significant departure from the original proposal lies in the timing of renewals, which now occur post-market phase. This adjustment aims to harmonize renewal prices with their market counterparts, ensuring a more consistent and equitable pricing model.
+
+#### Market Period (14 days)
```

Comment:
What's the motivation for 14 days of auctioning? What would be the shortest reasonable period for an auction?

Reply:
In the current design, I move all unsold cores to the instantanous market, therefore I wanted to give reasonable time for new-joiners to buy in the primary market. But I have no strong opinion on that, we could reduce the `MARKET_PERIOD` to 7 days and add the 7 days to the settlement phase.